docs/vscode-template/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "options": {"shell": {"executable": "bash", "args": ["-lic"]}},
+  "tasks": [
+    {
+      "label": "compile (maven)",
+      "type": "shell",
+      "command": [
+        "mvn",
+        "compile",
+        "-Dpmd.skip=true", // without pmd check
+        "-Dcheckstyle.skip=true" // without style check
+      ],
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "problemMatcher": []
+    },
+    {
+      "label": "package",
+      "type": "shell",
+      "command": ["mvn", "clean","package"],
+      "group": "build"
+    },
+    // {
+    //   "label": "verify",
+    //   "type": "shell",
+    //   "command": "mvn -B verify",
+    //   "group": "build"
+    // },
```

Comment:
Is there a reason to leave these in as comments versus deleting them?

Reply:
It was created as default by VSCode. I thought leaving them here in case someone wants it.
Maven verify is running the build pipeline to the end except installing the package.

Should I remove these lines?

Reply:
Let's rename the compile that skips checkstyle to debug-compile and add a tasks for full compile that runs all of the checks.

Reply:
Added Full compile tasks. And changed the name to `debug-compile (maven)`